### ISSUE TITLE
Problem with categories

### ISSUE THREAD

**Issue by Coryse2** [Fxdpt/skillcap#1]
Hi, 

I've tried to add new skills , I created types and categories before creating the skill -> no pb

but when I try to create a skill I can't choose a category (no pb for the type input) so I cant't validate my new skill

![Capture](https://user-images.githubusercontent.com/46875707/71548203-4317c880-29ab-11ea-8eb6-afaafa6f61af.JPG)

Maybe I made something wrong?

**Comment by Fxdpt** [comment]
Ok i think i have found something, when you go to show categories do you see your categories linked to types ? It seems that when you create a category and assign it types, when the form is submit the relation is not saved. But if you create a type and link it to existing categories it should work. I'm gonna fix this !

**Comment by Coryse2** [comment]
When I record a type I can see the related category, but when I create a catégory I don't see the related type

**Comment by Fxdpt** [comment]
yup, can't explain this ! all the entities are ok for me and form treatment is ok as well. I try to dump the form data when i create a category, and the object Category have the correct types assigned. It seems the problem appears when i persist a category
